@here/harp-mapview/lib/MapView.ts

```diff
@@ -1558,6 +1577,85 @@ export class MapView extends THREE.EventDispatcher {
         this.update();
     }
 
+    /**
+     * Returns the camera focus point position in geo-coordinates. If the camera points to an
+     * invalid position will return `null`.
+     */
+    get geoTarget() {
+        const target = MapViewUtils.rayCastWorldCoordinates(this, 0, 0);
+        if (target === null) {
+            throw new Error("Incorrect view properties, the target is null.");
```

Comment:
Why throwing an error here? This can easily happen if the user is modifying the three.js camera directly

Reply:
I asked for it :) it's because we have that throughout the code already, and MapView doesn't work if the target is in the void (the method to retrieve the tilt and azimuth doesn't know the target, so lookAt doesn't work)

Reply:
that's not acceptable. We should fix the places where MapView relies on that (I only know that the TiltClipPlaneEvaluator does) and don't add more.
It's totally fine if the map controls have this restriction but MapView should not have it.